app/lib/features/events/dialogs/create_event_sheet.dart

```diff
@@ -289,6 +279,21 @@ class _CreateEventSheetConsumerState extends ConsumerState<CreateEventSheet> {
     );
   }
 
+  Widget parentSpaceBuilder(
```

Comment:
Check if this being wrapped with ```Consumer```.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/855/commits/4239d5e28ce9904e7638301fb7f94b861d1481ae